app/src/main/java/a/a/a/dt.java

```diff
@@ -0,0 +1,254 @@
+                i4 = i + 1;
+                rs.a((Ts) rs.V.get(i), rs2);
+                i3 = i4;
+            } else if (((String) pair2.first).length() > 2) {
+                Object obj = pair2.first;
+                String substring = ((String) obj).substring(((String) obj).indexOf(".") + 1);
+                rs2 = new Rs(this.a, arrayList.indexOf(pair2) + 1, (String) pair2.second, kq.a(substring), kq.b(substring), "getArg");
+                viewGroup.addView(rs2);
+                rs.a((Ts) rs.V.get(i), rs2);
+                i3 = i + 1;
+            } else {
+                i3 = i;
+            }
+            i2++;
+            i = i3;
+        }
+        rs.k();
+        viewGroup2.removeAllViews();
+        i2 = rs.ka.size();
+        for (i = 0; i < i2; i++) {
+            View view = (View) rs.ka.get(i);
+            int a = ((String) rs.la.get(i)).equals("label") ? a((TextView) view) : 0;
+            if (view instanceof Rs) {
+                a = ((Rs) view).getWidthSum();
+            }
+            i3 = (int) (((float) a) + wB.a(this.a, 4.0f));
+            View imageView = new ImageView(this.a);
+            imageView.setImageResource(2131165831);
+            imageView.setScaleType(ScaleType.CENTER_INSIDE);
+            imageView.setPadding(0, (int) wB.a(this.a, 4.0f), 0, (int) wB.a(this.a, 4.0f));
+            imageView.setLayoutParams(new LayoutParams(i3, -1));
+            viewGroup2.addView(imageView);
+            if (i != 0 || this.g.getText().length() <= 0) {
+                imageView.setOnClickListener(new ct(this, arrayList, viewGroup2, viewGroup, rs, str));
+            } else {
+                imageView.setVisibility(4);
+                imageView.setEnabled(false);
+            }
+        }
+    }
+
+    public boolean a() {
+        return this.g.getText().toString().isEmpty() && this.l.size() == 0;
+    }
+
+    public boolean b() {
+        if (this.g.getText().toString().isEmpty()) {
+            bB.b(getContext(), xB.b().a(getContext(), 2131625494), 0).show();
+            return false;
+        } else if (this.o.b()) {
+            return true;
+        } else {
+            bB.b(getContext(), xB.b().a(getContext(), 2131625494), 0).show();
+            return false;
+        }
+    }
+
+    public Pair<String, String> getBlockInformation() {
+        String trim = this.g.getText().toString().trim();
+        return new Pair(ReturnMoreblockManager.injectMbType(trim, trim, getType()), ReturnMoreblockManager.injectMbType(this.p.T, trim, getType()));
+    }
+
+    public String getType() {
+        return ReturnMoreblockManager.getMbTypeFromRadioButton(this.radio_mb_type);
+    }
+
+    public void initRefresh() {
+        this.radio_mb_type.setOnCheckedChangeListener(new a(this));
+        radiobutton1.setOnCheckedChangeListener(new OnCheckedChangeListener() {
+			@Override
+			public void onCheckedChanged(RadioGroup radioGroup, int i) {
+				refresh(getType());
+			}
+		});

```

Comment:
Seems like these lines wouldn't compile, could you check them again?

Reply:
I updated it.Can you try again?